docs/docs/segment_nightscout.md

```diff
@@ -0,0 +1,60 @@
+---
+id: nightscout
+title: Nightscout
+sidebar_label: Nightscout
+---
+
+## What
+
+[Nightscout](http://www.nightscout.info/) (CGM in the Cloud) is an open source, DIY project that allows real time access to a CGM data via an HTTP REST API. It is used for secure remote viewing of blood sugar data from anywhere...including OhMyPosh segments on the command line!
+
+## Sample Configuration
+
+This example is using mg/dl, you'll want change the numbers for mmol. Your idea of "high" or "low" is different from others. You'll also want to think about your background and foreground colors. Don't use white text on a yellow background, for example.
+
+The foreground_templates example below could be set to just a single color, if that color is visible against any of your backgrounds. 
+
+```json
+{
+  "type": "nightscout",
+  "style": "diamond",
+  "foreground": "#ffffff",
+  "background": "#ff0000",
+  "background_templates": [
+    "{{ if gt .Sgv 150 }}#FFFF00{{ end }}",
+    "{{ if lt .Sgv 60 }}#FF0000{{ end }}",
+    "#00FF00"
```

Comment:
I'm confused, we want folks to have the ability to change colors, and this is three colors. What would we add to the background property?

Reply:
Here what it looks like at the moment:

```
                {
                    "type": "nightscout",
                    "style": "powerline",
                    "powerline_symbol": "",
                    "invert_powerline": false,
                    "foreground": "#000000",
                    "background": "#ff0000",
                    "background_templates": [
                      "{{ if gt .Sgv 150 }}#FFFF00{{ end }}",
                      "{{ if lt .Sgv 60 }}#FF0000{{ end }}",
                      "#00FF00"
                    ],          
                    "leading_diamond": "",
                    "trailing_diamond": "",
                    "properties": {
                      "url": "https://hanselsugar.herokuapp.com/api/v1/entries.json?count=1&token=ohmyposh-efebf7e3cb48d5d1",
                      "http_timeout": 2000,
                      "template": " {{.Sgv}}{{.TrendIcon}}"
                    }
                  },
```

<img width="430" alt="image" src="https://user-images.githubusercontent.com/2892/143140975-b2cea7b2-44fc-4680-9991-0a5c9eabc560.png">

Reply:
good question, using the following will result in the `background` never being used. It evaluates the templates sequentially and in case of no match it will always return `#00FF00` and never `#ff0000`. Adding `#ff0000` and removing it from the templates has the same result as `background` is the always the fallback color. But I can merge it like that.